app/index.jsx

```diff
@@ -280,6 +280,7 @@ class App extends Component {
                 granularity={this.state.granularity}
                 onSettingsChange={this.saveSettings}
                 isConnectionOpened={isConnectionOpened}
+                noTradingTimesForStartEpoch
```

Comment:
@nazaninreihani It'd be better to change the name to `shouldFetchTradingTimes` and default to `True`.
```suggestion
                shouldFetchTradingTimes
```

Reply:
yesss thanks, I couldn't think of a name for it I was hoping you suggest a name :-D